app/controllers/accounts/connected_accounts_controller.rb

```diff
@@ -16,7 +18,15 @@ def show
         sp_name: decorated_sp_session.sp_name,
         user: current_user,
         locked_for_session: pii_locked_for_session?(current_user),
+        change_email_available: change_email_available?,
```

Comment:
To me, part of the benefit of having a presenter like this is to help keep controllers lean by moving the logic into the presenter class, but in this case we're using the presenter as a pass-through and having the logic in the controller.

Instead, I might have expected that we could pass a value which is more readily available (`decorated_sp_session.requested_attributes` or `decorated_sp_session`) and do the logic in the presenter.